__tests__/app.test.js

```diff
@@ -26,6 +26,39 @@ describe("GET /api/categories", () => {
   });
 });
 
+describe("GET /api/reviews/:review_id", () => {
+    it("should return status 200, responds with a review object", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then((response) => {
+          const reviewObject = response.body.review[0]
+          const expectedReview = {
+            review_id: 2,
+              title: 'Jenga',
+      designer: 'Leslie Scott',
+      owner: 'philippaclaire9',
+      review_img_url:
+        'https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700',
+      review_body: 'Fiddly fun for all the family',
+      category: 'dexterity',
+      created_at: "2021-01-18T10:01:41.251Z",
+      votes: 5
+          }
+            expect(reviewObject).toEqual(expectedReview);
+          });
+        });
+    });
+    it("should return status 404 when reponding to invalid review_id in request", () => {
```

Comment:
great testing so far! You're just missing a 400 test for an endpoint like /api/reviews/banana